README.md

```diff
@@ -273,12 +159,6 @@ $ docker run \
 	ghcr.io/filecoin-station/core
 ```
 
-Inspect logs:
-
-```bash
-$ docker exec station ./bin/station.js logs
-```
-
```

Comment:
How about we keep this section and change the command to `docker logs station`. WDYT?

Reply:
LOL, it reminds me of the old acronym [RTFM](https://en.wikipedia.org/wiki/RTFM). It wasn't a very friendly way how to treat users.

I agree documenting Docker CLI is out of scope of this README.

At the same time, I think a paragraph showing Station Core users how to access their logs has more benefits than costs.

Anyhow, I am fine either way. You decide.

Reply:
While I would never say RTFM, I also wouldn't say it's outdated - a lot of projects follow this practice I think. If you feel strongly about this, I'm fine to add the comment, but I personally value minimalism in code projects over friendliness - the friendly parts can make it harder to scan the documentation or code for what you're really looking for. A project's tone shouldn't be harsh and too dry either, but minimalistic neutral works very well for me.